ironfish/src/telemetry/telemetry.ts

```diff
@@ -166,4 +166,34 @@ export class Telemetry {
       ],
     })
   }
+
+  submitNewBlockSeen(block: Block, seenAt: Date): void {
+    this.submit({
+      measurement: 'node',
+      name: 'propagation',
+      timestamp: seenAt,
+      tags: [
+        {
+          name: 'block_hash',
+          value: block.header.hash.toString('hex'),
+        },
+        {
+          name: 'block_sequence',
+          value: block.header.sequence.toString(),
+        },
+      ],
+      fields: [
+        {
+          name: 'block_timestamp',
+          type: 'integer',
+          value: block.header.timestamp.valueOf(),
+        },
+        {
+          name: 'block_sequence_field',
```

Comment:
Do we need the sequence as both a tag and field?

Reply:
It was used, but no longer. I removed it from the tags.